features/support/helpers/capybara_helpers.rb

```diff
@@ -12,11 +12,11 @@ module CapybaraHelpers
   ].join(', ').freeze
 
   def javascript_test?
-    Capybara.current_driver == Capybara.javascript_driver
+    Capybara.current_driver != Capybara.default_driver
```

Comment:
Now `:chrome` and `:headless_chrome` will act the same way, tests run with `:chrome` will be "javascript tests", which makes sense.

Reply:
Previously when setting `:chrome` as the js driver, this check was still valid. Or how to you use `:chrome` as the browser so this didn't work?

Reply:
Not really, because `current_driver == :crome`, and `javascript_driver == :headless_chrome` :shrug:

Reply:
So you set `@chrome` on scenarios?

Reply:
That's right. If I want to debug what's going on in the browser locally (see what elements are rendered, play around with the DOM etc.), I add `@chrome` on the scenario I'm testing, and it shows a chrome window.

Isn't it what everyone is doing? :sweat_smile: